### PR TITLE
Fixed Gqueries for investment table

### PR DESCRIPTION
**Pull request by jorisberkhout**

### CODE REVIEW

**Comment by jorisberkhout** [comment]
Part of https://github.com/quintel/etmodel/pull/2234

File: gqueries/output_elements/output_series/table_142_investments/total_future_in_investment_cost_table.gql [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
...t_cost_table),Q(industry_flexibility_p2g_electricity_future_in_investment_cost_table),Q(industry_other_food_burner_coal_future_in_investment_cost_table),Q(industry_other_food_burner_crude_oil_future_in_investment_cost_table),Q(industry_other_food_burner_network_gas_future_in_investment_cost_table),Q(industry_other_food_burner_wood_pellets_future_in_investment_cost_table),Q(industry_other_paper_burner_coal_future_in_investment_cost_table),Q(industry_other_paper_burner_crude_oil_future_in_investment_cost_table),Q(industry_other_paper_burner_network_gas_future_in_investment_cost_table),Q(industry_other_paper_burner_wood_pellets_future_in_investment_cost_table),Q(industry_chp_combined_cycle_gas_power_fuelmix_future_in_investment_cost_table),Q(industry_chp_engine_gas_power_fuelmix_future_in_investment_cost_table),Q(industry_chp_turbine_gas_power_fuelmix_future_in_investment_cost_table),Q(industry_chp_ultra_supercritical_coal_future_in_investment_cost_table),Q(other_burner_coal_future_in_investment_cost_table),Q(other_burner_crude_oil_future_in_investment_cost_table),Q(other_burner_network_gas_future_in_investment_cost_table),Q(other_burner_wood_pellets_future_in_investment_cost_table),)
```
Comment (ChaelKruip) [inline gqueries/output_elements/output_series/table_142_investments/total_future_in_investment_cost_table.gql]: There is a trailing comma in this summation.

File: gqueries/output_elements/output_series/table_142_investments/total_present_in_investment_cost_table.gql [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
...ndustry_flexibility_p2g_electricity_present_in_investment_cost_table),Q(industry_other_food_burner_coal_present_in_investment_cost_table),Q(industry_other_food_burner_crude_oil_present_in_investment_cost_table),Q(industry_other_food_burner_network_gas_present_in_investment_cost_table),Q(industry_other_food_burner_wood_pellets_present_in_investment_cost_table),Q(industry_other_paper_burner_coal_present_in_investment_cost_table),Q(industry_other_paper_burner_crude_oil_present_in_investment_cost_table),Q(industry_other_paper_burner_network_gas_present_in_investment_cost_table),Q(industry_other_paper_burner_wood_pellets_present_in_investment_cost_table),Q(industry_chp_combined_cycle_gas_power_fuelmix_present_in_investment_cost_table),Q(industry_chp_engine_gas_power_fuelmix_present_in_investment_cost_table),Q(industry_chp_turbine_gas_power_fuelmix_present_in_investment_cost_table),Q(industry_chp_ultra_supercritical_coal_present_in_investment_cost_table),Q(other_burner_coal_present_in_investment_cost_table),Q(other_burner_crude_oil_present_in_investment_cost_table),Q(other_burner_network_gas_present_in_investment_cost_table),Q(other_burner_wood_pellets_present_in_investment_cost_table),)
```
Comment (ChaelKruip) [inline gqueries/output_elements/output_series/table_142_investments/total_present_in_investment_cost_table.gql]: Idem here.

**Comment by jorisberkhout** [comment]
I have fixed the gqueries that calculate the total of the investment table to make use of the correct gqueries for combi technologies. This avoids double counting.

I have tested the new table pretty thoroughly and everything behaves as expected.

**Pull request merged**